### DISCUSSION TITLE
Invalid credentials

### DISCUSSION THREAD

**Discussion by droopified** [photoprism/photoprism#2803] [General]
ok sooo out of nowhere, im getting Invalid credentials when logging in. absolutely nothing has been changed to photoprism.  i dont see any problems in the log either. tried changing the Initial Admin Password in docker but still nothing. When i make the changes to add the variable to make it public, everything is working. I changed the password to 'password' for testing and it just doesn't let me in. I'm able to sign in to the adminer and see the users thinking maybe the username changed but it's still 'admin' not 'Admin'. I'm using unRAID with the preview release.

**Comment by graciousgrey** [comment]
--> https://docs.photoprism.app/release-notes/#development-preview

> Session and user management have been re-implemented. If you are upgrading from the last preview, you may need to run the "photoprism users reset" [command in a terminal](https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface) after the upgrade to recreate the new database tables. They are not compatible with the last preview, which is why we have stopped releasing preview builds until they are stable enough.

**Reply by droopified** [reply]
\u@:\w$ photoprism users reset
INFO[2022-10-26T16:40:07-05:00] PhotoPrism® needs your support!              
INFO[2022-10-26T16:40:07-05:00] Visit https://photoprism.app/membership to learn more. 
? Reset the user database to a clean state?? [y/N] y█
INFO[2022-10-26T16:41:00-05:00] the user database has been recreated and is now in a clean state 
INFO[2022-10-26T16:41:00-05:00] closed database connection                   
\u@:\w$ 

restarted the container but still unable to connect

edit: i changed the password again and restarted the container and now im able to login. thanks!

**Reply by lastzero** [reply]
Yes, after the reset you need to restart or manually migrate so an amin user exists or is migrated from the old table.
